src/wazuh_modules/syscollector/sysInfoWin.cpp

```diff
@@ -96,6 +349,45 @@ static std::string parseRawSmbios(const BYTE* rawData, const DWORD rawDataSize)
     return serialNumber;
 }
 
+static std::string processName(PROCESSENTRY32 processEntry,
```

Comment:
can be const?

Reply:
add a method like bool hasCmd or isSystemProcess to check if the cmd should be added to the json instead of using execPath string value.